src/main/webapp/css/main.css

```diff
@@ -56,3 +56,11 @@ input {
 button {
   font-size: 18px;
 }
+
+#previewBox {
```

Comment:
this is from when i was going to add a preview box. i'm still interested in implementing that later, maybe for the open project, but it isn't totally relevant to the functionality of this pr

Reply:
that's a great idea!